libafl/src/inputs/bytes.rs

```diff
@@ -24,6 +27,8 @@ pub struct BytesInput {
 }
 
 impl Input for BytesInput {
+    const NAME: &'static str = "BytesInput";
```

Comment:
I think here type_name can be used instead of an associated const https://doc.rust-lang.org/std/any/fn.type_name.html

Reply:
Actually, this needs to be an associated const because type_name is not const stable.

Reply:
> The exact contents and format of the string returned are not specified [...] amongst the strings that type_name::<Option<String>>() might return are "Option<String>" and "std::option::Option<std::string::String>".
 
Could be an issue, probably a NAME field is safer

Reply:
maye it should be called `TYPE` else it's confusing with our `named` trait(?)